tests/end-to-end/director/examplesaver.go

```diff
@@ -4,24 +4,26 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path"
 	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
-func saveQueryInExamples(t *testing.T, query string, exampleName string) {
+func saveQueryInExamples(t *testing.T, query string, exampleDirectory string, exampleName string) {
```

Comment:
Hmm, in most cases exampleName = exampleDirectory.

I would create two functions with shared logic:
```
func saveOperationInExamples(t *testing.T, query string, exampleName string)
func saveOperationInExamplesWithCustomDir(t *testing.T, query string, exampleName string, customDirectory string)
```

By default, when customDirectory is not provided, you will create example in `example-name/example-name.graphql` file.

Reply:
Then, you won't need to have that much const in tests